Snakemake/workflow/profile/config.yaml

```diff
@@ -2,11 +2,11 @@ restart-times: 0
 use-conda: true
 #jobscript: "jobscript.sh"
 #cluster-config: "cluster.json"
-jobname: "{rule}.{jobid}"
-#drmaa: " -P roslin_HighlanderLab -cwd -l h_vmem={cluster.mem} -l h_rt={cluster.time} -pe sharedmem {cluster.threads}"
+#jobname: "{rule}.{jobid}"
+#drmaa: " -cwd -l h_vmem={cluster.mem} -l h_rt={cluster.time} -pe sharedmem {cluster.threads}"
 rerun-incomplete: true
 rerun-triggers: mtime
 cores: 99 # how many jobs you want to submit to your cluster queue
 local-cores: 1
 keep-going: false
-
+#-P roslin_HighlanderLab
```

Comment:
Why did you remove this?

Reply:
I am not sure if that work on the wwn. But I'll put it back because we are only submitting on normal nodes anyway.

Reply:
I am using this on normal nodes so I think it would be good if it's in there :)